appcheck/firebase-appcheck-interop/firebase-appcheck-interop.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
 }
 
 firebaseLibrary {
+    libraryGroup "appcheck"
```

Comment:
Last release interop was not released

https://maven.google.com/web/index.html?q=appcheck#com.google.firebase:firebase-appcheck-interop 

vs

https://maven.google.com/web/index.html?q=appcheck#com.google.firebase:firebase-appcheck

we should double check with the appcheck team that this is actually intended:

1) If yes, then no need to add it to the library group
2) If no, we need to release it on its own